ci-operator/step-registry/openshift/release/fips/openshift-release-fips-commands.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Get major.minor from payload url
+major_minor=$(echo "${RELEASE_IMAGE_LATEST}" | awk -F':' '{print $2}' | awk -F'.' '{print $1"."$2}')
```

Comment:
You should probably defend against digest based pullspecs.  Technically, these are both valid values and point to the same release:

`registry.ci.openshift.org/ocp/release:4.16.0-0.nightly-2024-04-01-213440`
and
`registry.ci.openshift.org/ocp/release@sha256:2ff463cd4c32392e4318a37892934d4aaee6475393c9f3c3afff14b55bb94632`

Reply:
So when a nightly or promoted payload is created, which value will be passed? Is there a different scenario where the digest will be passed?

Reply:
I can only assure you that what comes out of the `release-controller` is a `Tag.Name` pullSpec.  However, all we are really doing is handing a Prowjob definition to Kubernetes.  What happens from that point until it gets to your step could literally be anything.  (Not saying it will, but it could and it's better to have an error that states exactly why it failed vs some crazy error from multiple piped `awk` commands).

Reply:
added in https://github.com/openshift/release/pull/50497/commits/8e84f777308a0ce05012f088440fcbfcd1178982